tests/config_test.go

```diff
@@ -359,21 +361,23 @@ var _ = Describe("[Serial][rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][le
 				expectedOutputSecret := "adminredhat"
 
 				By("Running VMI")
-
-				vmi := tests.NewRandomVMIWithEphemeralDiskHighMemory(
-					cd.ContainerDiskFor(
-						cd.ContainerDiskFedoraTestTooling))
-				tests.AddConfigMapDisk(vmi, configMapName, configMapName)
-				tests.AddSecretDisk(vmi, secretName, secretName)
-				tests.AddConfigMapDiskWithCustomLabel(vmi, configMapName, "random1", "configlabel")
-				tests.AddSecretDiskWithCustomLabel(vmi, secretName, "random2", "secretlabel")
+				vmi := libvmi.NewFedora(withConfigMap(configMapName, configMapName),
+					withSecret(secretName, secretName),
+					withLabelledConfigMap(configMapName, "random1", "configlabel"),
+					withLabelledSecret(secretName, "random2", "secretlabel"))
 
 				// Ensure virtio for consistent order
 				for i := range vmi.Spec.Domain.Devices.Disks {
-					vmi.Spec.Domain.Devices.Disks[i].Disk = &v1.DiskTarget{Bus: "virtio"}
+					vmi.Spec.Domain.Devices.Disks[i].DiskDevice = v1.DiskDevice{
+						Disk: &v1.DiskTarget{
+							Bus: v1.DiskBusVirtio,
+						},
+					}
 				}
 
-				vmi = tests.RunVMIAndExpectLaunch(vmi, 90)
+				vmi = tests.RunVMIAndExpectLaunch(vmi, 60)
```

Comment:
I would split this change to a separate commit and add a comment so future us know why we pick this magic number...